lib/minimap-find-and-replace-binding.js

```diff
@@ -49,16 +49,17 @@ class MinimapFindAndReplaceBinding {
 	}
 
 	clear () {
-		let id
-		for (id in this.subscriptionsByMarkerId) {
-			const sub = this.subscriptionsByMarkerId[id]
-			sub.dispose()
+		const subscriptionsByMarkerIdKeys = Object.keys(this.subscriptionsByMarkerId)
+		for (let i = 0, len = subscriptionsByMarkerIdKeys.length; i < len; i++) {
```

Comment:
Is this for loop really faster than for in? I feel like V8 would be able to optimize a for in loop a lot easier.

Reply:
Yes, it is faster. 
https://github.com/aminya/typescript-optimization

Reply:
I will just replace it with Map. Object should not be used in these cases.